src/Microsoft.ApiDesignGuidelines.Analyzers/Microsoft.ApiDesignGuidelines.Analyzers.md

```diff
@@ -190,6 +298,26 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182243.aspx](http://msdn.microsoft.com/library/ms182243.aspx)
 
+### CA1716: Identifiers should not match keywords ###
+
+A namespace name or a type name matches a reserved keyword in a programming language. Identifiers for namespaces and types should not match keywords that are defined by languages that target the common language runtime.
+
+Category: Naming
+
+Severity: Warning
+
+Help: [https://msdn.microsoft.com/en-us/library/ms182248.aspx](https://msdn.microsoft.com/en-us/library/ms182248.aspx)
+
+### CA1717: Only FlagsAttribute enums should have plural names ###
+
+Naming conventions dictate that a plural name for an enumeration indicates that more than one value of the enumeration can be specified at the same time.
+
```

Comment:
This description has a very different tone than the one for CA1714. "Naming conventions dictate..." is kinda harsh. I would start with the text for CA1714, which I like, and reverse it for this case, maybe like this:

>  The name of a public enumeration ends in "s", but it does not have the `System.FlagsAttribute` attribute. The plural type name suggests that more than one value can be specified. If this is the case, add the `System.FlagsAttribute` attribute. If this is not the case, use a name that is not a plural.